zerver/lib/email_notifications.py

```diff
@@ -138,7 +138,17 @@ def fix_plaintext_image_urls(content: str) -> str:
         # with a simple hyperlink.
         return re.sub(r"\[(\S*)\]\((\S*)\)", r"\2", content)
 
-    def build_message_payload(message: Message) -> Dict[str, str]:
+    def append_sender_to_message(plain: str, html: str, sender: str) -> Tuple[str, str]:
+        plain = "{}: {}".format(sender, plain)
+
+        sender_tag = "<b>{}</b>:".format(sender)
+        if html[0:3] == "<p>":
+            html = "<p>{} {}".format(sender_tag, html[3:])
+        else:
+            html = "{}<br>{}".format(sender_tag, html)
```

Comment:
Do we have any test cases that cover this code path?

Reply:
Added tests for this.